Cargo.toml

```diff
@@ -101,12 +101,12 @@ foundry-evm-fuzz = { path = "crates/foundry/evm/fuzz" }
 foundry-evm-traces = { path = "crates/foundry/evm/traces" }
 
 # solc & compilation utilities
-foundry-block-explorers = { version = "=0.11.0", default-features = false }
-foundry-compilers = { version = "=0.14.1", features = ["full"] }
+foundry-block-explorers = { version = "0.18.0", default-features = false }
+foundry-compilers = { version = "0.17.3", default-features = false }
```

Comment:
Is there a reason to keep these as `=` version specifiers?

Reply:
I pinned these, because later patch versions of those minors wouldn't work with some other dependencies. If it works now, I think it's ok to no longer pin to patch version.